packages/cli/src/commands/push.ts

```diff
@@ -32,11 +32,13 @@ const register: (program: any) => any = (program) => program
   .option('-d, --deploy-dependencies', 'deploys dependencies to the network if there is no existing deployment')
   .option('--reset', 'redeploys all contracts (not only the ones that changed)')
   .option('-f, --force', 'ignores validation errors and deploys contracts')
+  .option('--proxy-admin', 'eagerly deploys the project\'s proxy admin (if not deployed yet on this network)')
+  .option('--proxy-factory', 'eagerly deploys the project\'s proxy factory (if not deployed yet on this network)')
```

Comment:
WDYT about being a bit more explicit with naming here, and change both flags to `--deploy-proxy-admin` and `--deploy-proxy-factory` ?

I think it will be so much clear!
```sh
zos push --network local --deploy-proxy-admin --deploy-proxy-factory
```
It's a personal opinion though.

Reply:
Like the idea!